src/main/java/org/osc/manager/ism/entities/VSSDevice.java

```diff
@@ -29,24 +29,22 @@
 public class VSSDevice {
```

Comment:
vs should have one to many relationships to SG and SGI.

Reply:
VSSDevice will be removed soon, instead use DeviceEntity.